src/tests/ftest/launch.py

```diff
@@ -897,6 +901,74 @@ def archive_logs(avocado_logs_dir, test_yaml, args):
     spawn_commands(host_list, "; ".join(commands), 900)
 
 
+def send_notification(hosts, subject, msg, attachment, emails):
+    """Send email notification to provided emails with given message.
+
+    Args:
+        host (list): hosts to perform command in.
+        msg (str): message to be sent.
+        attachment (str): attachment to send on the email.
+        emails (list): list of email addresses to send message to.
+    """
+    mail_cmd = "mail -s \"{}\" -a {} {} <<< \"{}\"".format(
+        subject, attachment, ",".join(emails), msg)
+    spawn_commands(hosts, mail_cmd, 30)
+
+
+def get_log_size(test_yaml, test_file, args, size_limit=2**33):
```

Comment:
How was up to 8GB determined to be an acceptably large log file?

Reply:
I haven't found a specific metric to base this number on. I've seen that most logs (from digging in the jenkins artifacts) do not generally exceed much over maybe close to 1gb and sometimes a bit more. I thought that by assigning a somewhat arbitrary number to the limit  we can learn to calibrate it as more investigation is done with the use of the email notifications.

Reply:
Good plan of action.  However if most logs are less than 8GB but are say, ~7GB, you are not going to get notified about those and so we'd still have most logs being quite large at 7GB.

I'd say we should be setting an arbitrarily low figure so that we get a lot of data on log sizes upon which to set the upper limit and possibly investigate for *too-large*ness.

Reply:
Good point, agreed. In which case I'll bring it down to 1GB to start getting some data on it.